### PR TITLE
Updated readthedocs configuration - moved to build.os #none

### PR DESCRIPTION
**Pull request by kamaleybov**
# TL;DR
Upgrades readthedocs configuration to use `build.os` setting.

This is a required change in order to continue using readthedocs, as they are deprecating configurations not using `build.os` and started failing the builds: https://blog.readthedocs.com/use-build-os-config/

## Type
- [x] Bug Fix
- [ ] Feature
- [ ] Plugin

(Not really a bug fix, rather an upgrade.)

## Are all requirements met?

- [x] Code completed
- [ ] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
See TLDR.

## Tracking Issue
_NA_

## Follow-up issue
_NA_

### CODE REVIEW

**Review by eapolinario — APPROVED** [review]

**Review by pingsutw — APPROVED** [review]

**Comment by davidmirror-ops** [comment]
Thank you @kamaleybov!. I was about to mention this problem that was breaking my PR
